.github/workflows/downloads-count.yaml

```
@@ -0,0 +1,72 @@
+# Launch bot
+name: Downloads Count
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+
+  get-all-releases-count:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/github-script@v6
+        id: process-downloads
+        with:
+          script: |
+            const resp = await github.paginate('GET /repos/{owner}/{repo}/releases{?per_page,page}', {
+                owner: 'containers',
+                repo: 'podman-desktop',
+                per_page: 100
+            }, (response) => response.data.map(({ tag_name, assets }) => ({ tag_name, assets })));
+
+            const releases = resp;
+
+            // Create a new json array which keeps tag_name and assets from result
+            //const releases = resp.data.map(({ tag_name, assets }) => ({ tag_name, assets }))
```

```suggestion
            // Create a new json array which keeps tag_name and assets from result
            //const releases = resp.data.map(({ tag_name, assets }) => ({ tag_name, assets }))
```

looks like comments can be removed